cla_public/apps/contact/fields.py

```diff
@@ -76,7 +80,9 @@ def process_formdata(self, valuelist):
 
     @classmethod
     def _format(cls, value):
-        return '{:%Y%m%d}'.format(value)
+        if value and not isinstance(value, basestring):
```

Comment:
No wait, bloody folds. It's in `DayChoiceField`, so it's definitely a datetime or date